README.md

```diff
@@ -206,7 +245,7 @@ The purpose of this option is as follows.  liboqs master branch and liboqs nist-
 	- `make clean`
 	- `make -j`
 	- `make install`
-3. Run `ssh` with `ecdh-nistp384-oqsdefault-sha384@openquantumsafe.org ` or `oqsdefault-sha384@openquantumsafe.org` for the `KexAlgorithms` option.
+3. Run `ssh` with `ecdh-nistp384-oqsdefault-sha384@openquantumsafe.org ` or `oqsdefault-sha384@openquantumsafe.org` for the `KexAlgorithms` option, or +++
```

Comment:
"or +++" -- presumably more text to add here?

Reply:
Yes, missing text; I'll add it.